README.md

```diff
@@ -63,11 +63,50 @@ Example Alert:
       description: Filebeat queue is {{printf "%.0f" $value}} and growing
 ```
 
+## Kubernetes example for exposed metric with non-default port
```

Comment:
Do you really sure that single cli argument like setting a port deserves explicit Readme section and whole k8s deployment example? Maybe it is enough to just document the argument in the list of arguments, because it is a typical case?

Reply:
Removed example and re-organized readme.